frontend/packages/console-shared/src/utils/sample-utils.ts

```diff
@@ -60,6 +79,17 @@ const defaultProjectAccessRoles: ProjectAccessRoles = {
   availableClusterRoles: ['admin', 'edit', 'view'],
 };
 
+const defaultDisabledPerspectives: PerspectiveOptions = {
+  id: 'admin',
+  requiresAccessReview: [
+    {
+      group: 'rbac.authorization.k8s.io',
+      resource: 'clusterroles',
+      verb: 'list',
+    },
+  ],
+};
```

Comment:
can you fetch all the available perspectives from the console perspective extensions like how you did for the sub-catalog types ?

And see the latest API here : https://github.com/openshift/enhancements/pull/1205

@lokanandaprabhu @jerolimov Should the code snippet show `state: Enabled` for all the perspectives?

Reply:
We have no other option if we want to show all perspective names in the snippet, right? Sounds good to me.